views/chorals.pug

```diff
@@ -30,3 +30,9 @@ block content
           td
             button.btn.btn-danger.delete-choral-btn(data-choral-id=choral.choralId)
               span.glyphicon.glyphicon-trash
+
+    script.
+      $('.choral-row').on("click", function() {
+        document.location = $(this).data('href');
```

Comment:
This seems like a weird way to do this. why not just make a link. Especially since this means that the cursor isnt going to look like a link when you hover over the row. Better yet make a button in a new column that says "View Live Data" or something.